GTFS/GTFSWriter.cs

```diff
@@ -346,6 +346,7 @@ protected virtual void Write(IGTFSTargetFile file, IEnumerable<FareAttribute> en
                         data[3] = "payment_method";
                         data[4] = "transfers";
                         data[5] = "transfer_duration";
+                        data[6] = "agency_id";
```

Comment:
maybe put this one in data[5]. it fits better with the ordering of the fields in the gtfs spec: https://developers.google.com/transit/gtfs/reference/#fare_attributestxt

Reply:
yeah, you only need to do that field switch here